lighthouse-cli/server/server.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
is there a more descriptive directory name we can use here since this is going to become specifically the performance experiments server?